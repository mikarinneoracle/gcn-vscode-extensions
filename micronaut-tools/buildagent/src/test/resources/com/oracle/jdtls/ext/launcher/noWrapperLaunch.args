/ext/redhat.java-1.19.0-linux-x64/jre/17.0.7-linux-x86_64/bin/java
-XX:+ShowCodeDetailsInExceptionMessages
-cp
/space/tmp/vsc2/src3/demo11/oci/target/classes:.m2/repository/io/micronaut/micronaut-inject/3.8.5/micronaut-inject-3.8.5.jar:.m2/repository/org/slf4j/slf4j-api/1.7.36/slf4j-api-1.7.36.jar:.m2/repository/javax/annotation/javax.annotation-api/1.3.2/javax.annotation-api-1.3.2.jar:.m2/repository/jakarta/inject/jakarta.inject-api/2.0.1/jakarta.inject-api-2.0.1.jar:.m2/repository/io/micronaut/micronaut-core/3.8.5/micronaut-core-3.8.5.jar:.m2/repository/org/yaml/snakeyaml/1.33/snakeyaml-1.33.jar:.m2/repository/io/micronaut/micronaut-validation/3.8.5/micronaut-validation-3.8.5.jar:.m2/repository/io/micronaut/micronaut-core-reactive/3.8.5/micronaut-core-reactive-3.8.5.jar:.m2/repository/org/reactivestreams/reactive-streams/1.0.4/reactive-streams-1.0.4.jar:.m2/repository/javax/validation/validation-api/2.0.1.Final/validation-api-2.0.1.Final.jar:.m2/repository/io/projectreactor/reactor-core/3.5.0/reactor-core-3.5.0.jar:.m2/repository/io/micronaut/micronaut-http-client/3.8.5/micronaut-http-client-3.8.5.jar:.m2/repository/io/micronaut/micronaut-runtime/3.8.5/micronaut-runtime-3.8.5.jar:.m2/repository/io/micronaut/micronaut-aop/3.8.5/micronaut-aop-3.8.5.jar:.m2/repository/io/micronaut/micronaut-context/3.8.5/micronaut-context-3.8.5.jar:.m2/repository/io/micronaut/micronaut-http-client-core/3.8.5/micronaut-http-client-core-3.8.5.jar:.m2/repository/io/micronaut/micronaut-websocket/3.8.5/micronaut-websocket-3.8.5.jar:.m2/repository/io/micronaut/micronaut-http-netty/3.8.5/micronaut-http-netty-3.8.5.jar:.m2/repository/io/micronaut/micronaut-buffer-netty/3.8.5/micronaut-buffer-netty-3.8.5.jar:.m2/repository/io/netty/netty-codec-http2/4.1.87.Final/netty-codec-http2-4.1.87.Final.jar:.m2/repository/io/netty/netty-handler/4.1.87.Final/netty-handler-4.1.87.Final.jar:.m2/repository/io/netty/netty-resolver/4.1.87.Final/netty-resolver-4.1.87.Final.jar:.m2/repository/io/netty/netty-transport-native-unix-common/4.1.87.Final/netty-transport-native-unix-common-4.1.87.Final.jar:.m2/repository/io/netty/netty-handler-proxy/4.1.87.Final/netty-handler-proxy-4.1.87.Final.jar:.m2/repository/io/netty/netty-common/4.1.87.Final/netty-common-4.1.87.Final.jar:.m2/repository/io/netty/netty-buffer/4.1.87.Final/netty-buffer-4.1.87.Final.jar:.m2/repository/io/netty/netty-transport/4.1.87.Final/netty-transport-4.1.87.Final.jar:.m2/repository/io/netty/netty-codec/4.1.87.Final/netty-codec-4.1.87.Final.jar:.m2/repository/io/netty/netty-codec-socks/4.1.87.Final/netty-codec-socks-4.1.87.Final.jar:.m2/repository/io/micronaut/micronaut-http-server-netty/3.8.5/micronaut-http-server-netty-3.8.5.jar:.m2/repository/io/micronaut/micronaut-http-server/3.8.5/micronaut-http-server-3.8.5.jar:.m2/repository/io/micronaut/micronaut-router/3.8.5/micronaut-router-3.8.5.jar:.m2/repository/io/netty/netty-codec-http/4.1.87.Final/netty-codec-http-4.1.87.Final.jar:.m2/repository/io/micronaut/micronaut-jackson-databind/3.8.5/micronaut-jackson-databind-3.8.5.jar:.m2/repository/io/micronaut/micronaut-jackson-core/3.8.5/micronaut-jackson-core-3.8.5.jar:.m2/repository/io/micronaut/micronaut-json-core/3.8.5/micronaut-json-core-3.8.5.jar:.m2/repository/com/fasterxml/jackson/core/jackson-core/2.14.0/jackson-core-2.14.0.jar:.m2/repository/com/fasterxml/jackson/core/jackson-annotations/2.14.0/jackson-annotations-2.14.0.jar:.m2/repository/com/fasterxml/jackson/core/jackson-databind/2.14.1/jackson-databind-2.14.1.jar:.m2/repository/com/fasterxml/jackson/datatype/jackson-datatype-jdk8/2.14.0/jackson-datatype-jdk8-2.14.0.jar:.m2/repository/com/fasterxml/jackson/datatype/jackson-datatype-jsr310/2.14.0/jackson-datatype-jsr310-2.14.0.jar:.m2/repository/io/micronaut/data/micronaut-data-jdbc/3.9.6-oracle-00001/micronaut-data-jdbc-3.9.6-oracle-00001.jar:.m2/repository/io/micronaut/data/micronaut-data-runtime/3.9.6-oracle-00001/micronaut-data-runtime-3.9.6-oracle-00001.jar:.m2/repository/io/micronaut/data/micronaut-data-model/3.9.6-oracle-00001/micronaut-data-model-3.9.6-oracle-00001.jar:.m2/repository/io/micronaut/data/micronaut-data-tx/3.9.6-oracle-00001/micronaut-data-tx-3.9.6-oracle-00001.jar:.m2/repository/javax/transaction/javax.transaction-api/1.3/javax.transaction-api-1.3.jar:.m2/repository/io/micronaut/micronaut-http/3.8.5/micronaut-http-3.8.5.jar:.m2/repository/io/micronaut/sql/micronaut-jdbc/4.7.2/micronaut-jdbc-4.7.2.jar:.m2/repository/io/micronaut/flyway/micronaut-flyway/5.4.1-oracle-00001/micronaut-flyway-5.4.1-oracle-00001.jar:.m2/repository/org/flywaydb/flyway-core/8.5.13/flyway-core-8.5.13.jar:.m2/repository/io/micronaut/micronaut-management/3.8.5/micronaut-management-3.8.5.jar:.m2/repository/io/micronaut/sql/micronaut-jdbc-hikari/4.7.2-oracle-00001/micronaut-jdbc-hikari-4.7.2-oracle-00001.jar:.m2/repository/com/zaxxer/HikariCP/4.0.3/HikariCP-4.0.3.jar:.m2/repository/jakarta/annotation/jakarta.annotation-api/2.1.1/jakarta.annotation-api-2.1.1.jar:.m2/repository/ch/qos/logback/logback-classic/1.2.11/logback-classic-1.2.11.jar:.m2/repository/ch/qos/logback/logback-core/1.2.11/logback-core-1.2.11.jar:.m2/repository/mysql/mysql-connector-java/8.0.30/mysql-connector-java-8.0.30.jar:.m2/repository/com/google/protobuf/protobuf-java/3.21.12/protobuf-java-3.21.12.jar:.m2/repository/org/flywaydb/flyway-mysql/8.5.13/flyway-mysql-8.5.13.jar:/space/tmp/vsc2/src3/demo11/lib/target/classes
com.example.Main
normal
arg space
arg"quote
