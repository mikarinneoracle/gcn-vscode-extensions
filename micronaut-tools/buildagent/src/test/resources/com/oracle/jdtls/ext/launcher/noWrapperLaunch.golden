-Dexec.executable=/ext/redhat.java-1.19.0-linux-x64/jre/17.0.7-linux-x86_64/bin/java
-Dexec.args=-XX:+ShowCodeDetailsInExceptionMessages --class-path %classpath ${exec.mainClass} normal "arg space" "arg\"quote"
-Dexec.mainClass=com.example.Main
org.codehaus.mojo:exec-maven-plugin:3.1.0:exec
